cassis/xmi.py

```diff
@@ -83,9 +83,15 @@ def _parse_annotation(self, typesystem: TypeSystem, elem):
 
         # Map the xmi:id attribute to xmiID
         attributes['xmiID'] = int(attributes.pop('{http://www.omg.org/XMI}id'))
-        attributes['begin'] = int(attributes['begin'])
-        attributes['end'] = int(attributes['end'])
-        attributes['sofa'] = int(attributes['sofa'])
+
```

Comment:
Do you also support arbitrary feature structures in the XMI, in particular such that do not have begin/end, i.e. that inherit directly from `TOP` or `AnnotationBase` - for example the DKPro Core `CoreferenceChain` ([link](https://dkpro.github.io/dkpro-core/releases/1.10.0/docs/typesystem-reference.html#type-de.tudarmstadt.ukp.dkpro.core.api.coref.type.CoreferenceChain))?

Reply:
Yes, I just added support for that, see https://github.com/dkpro/dkpro-cassis/blob/master/tests/test_files/typesystems/typesystem_with_inheritance.xml ,